src/mrack/providers/aws.py

```diff
@@ -83,6 +83,12 @@ async def validate_hosts(self, hosts):
 
             try:
                 aws_image = self.ec2.Image(req_img)
+                if not aws_image:  # if user has no rights to use ami None is returned
+                    raise ValidationError(
+                        f"{self.dsp_name}: Used credentials for user who is "
```

Comment:
Sorry for not writing it the first time. What about  a message: "Image {req_imd} does not exist or you don't have permissions to read it."? It's shorter, says the same thing + mentions the possibility of the image not existing.

Reply:
This error is being handled by: ` except ClientError as image_err:`

Reply:
Or... are you refering to some edge case?

Reply:
So, this exception(`ValidationError`) will never be raised if image doesn't exist?

Reply:
Yes that's what I have encountered.

Reply:
There will be an exception with message that ami is invalid I hope i am not mistaken.

Reply:
Btw, don't wait for me on finishing the PR. I'm leaving the rest on you as my main point was addressed. 

Though, I'd still recommend to use a simple error message like: `You don't have permissions to use image: {req_img}` I.e. `Used credentials for user ` == `you`